src/tir/schedule/analysis.h

```diff
@@ -442,6 +442,79 @@ bool CanComputeAt(const ScheduleState& self, const StmtSRef& block_sref, const S
 bool CanReverseComputeAt(const ScheduleState& self, const StmtSRef& block_sref,
                          const StmtSRef& loop_sref, bool preserve_unit_loops);
 
+/******** Tensorization Related ********/
+
+using ExprComparator = ExprFunctor<bool(const PrimExpr& n, const PrimExpr& other)>;
```

Comment:
Shall we move this into `src/tir/schedule/ir_comparator.h` and the impl to a corresponding cc file? The reason I'm considering this is that it's relatively independent but takes huge chunk of code